File: Insurance-2306226864-fe/src/enums/orderedPlanStatus.enum.ts
export enum OrderedPlanStatusEnum {
  ORDERED = 'ORDERED',
  PAID = 'PAID',
  WAITING_FOR_REVIEW = 'WAITING_FOR_REVIEW',
  CLAIMED = 'CLAIMED',
  REJECTED = 'REJECTED',
  EXPIRED = 'EXPIRED'
}

export const OrderedPlanStatusLabels: Record<OrderedPlanStatusEnum, string> = {
  [OrderedPlanStatusEnum.ORDERED]: 'Ordered',
  [OrderedPlanStatusEnum.PAID]: 'Paid',
  [OrderedPlanStatusEnum.WAITING_FOR_REVIEW]: 'Waiting for Review',
  [OrderedPlanStatusEnum.CLAIMED]: 'Claimed',
  [OrderedPlanStatusEnum.REJECTED]: 'Rejected',
  [OrderedPlanStatusEnum.EXPIRED]: 'Expired'
}

export const OrderedPlanStatusColors: Record<OrderedPlanStatusEnum, string> = {
  [OrderedPlanStatusEnum.ORDERED]: 'bg-gray-100 text-gray-800',
  [OrderedPlanStatusEnum.PAID]: 'bg-blue-100 text-blue-800',
  [OrderedPlanStatusEnum.WAITING_FOR_REVIEW]: 'bg-yellow-100 text-yellow-800',
  [OrderedPlanStatusEnum.CLAIMED]: 'bg-green-100 text-green-800',
  [OrderedPlanStatusEnum.REJECTED]: 'bg-red-100 text-red-800',
  [OrderedPlanStatusEnum.EXPIRED]: 'bg-gray-100 text-gray-600'
}

File: src/main/java/apap/ti/_5/Insurance_2306226864_be/enums/OrderedPlanStatusEnum.java
package apap.ti._5.Insurance_2306226864_be.enums;

public enum OrderedPlanStatusEnum {
    ORDERED("Ordered"),
    PAID("Paid"),
    WAITING_FOR_REVIEW("Waiting for Review"),
    CLAIMED("Claimed"),
    REJECTED("Rejected"),
    EXPIRED("Expired");

    private final String displayName;

    OrderedPlanStatusEnum(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }

    @Override
    public String toString() {
        return displayName;
    }
}